src/bots/googleChat.js

```diff
@@ -50,56 +50,83 @@ const setWebhookUrl = (newWebhookUrl) => {
   webhookUrl = newWebhookUrl;
 }
 
+/**
+ * Action button base url to redirect when an incident has actions that redirect to inOrbit
+ */
+let inorbitBaseUrl;
+function setInorbitBaseUrl(baseUrl) {
+  inorbitBaseUrl = baseUrl;
+}
+
 /**
  * Creates and styles the google chat message card
  * Returns the card with detail of the alert received in params
  * https://developers.google.com/chat/reference/message-formats/cards
  */
-const createGoogleChatCards = ({ severity, message, status, name, date, label, id }) => [
-  {
-    "header": {
-      "title": `${status} alert from InOrbit</b>`,
-      "subtitle": `Robot name: ${name}`
+const createGoogleChatCards = ({ severity, message, status, name, date, label, id, actions }) => {
+  // Parse actions into text buttons actions
```

Comment:
I suggest moving the logic to parse actions and convert them to buttons in `index.js`, around Line 60 which is where the parsing of InOrbit input is done - including the link to the more detailed documentation.
Leave the responsibility of this method to be only about creating the card, without having to know anything at all about InOrbit. It should only know that it needs to append some buttons.